README.md

```diff
@@ -1,68 +1,124 @@
 
 The attribute is configurable within the `plugin_specific_configuration` for the indexing rule engine plugin.
 
 ### Resource Metadata
 
-An administrator may wish to restrict indexing activities to particular resources, for example when automatically ingesting data.  Should a storage resource be at the edge, that resource may not be appropriate for indexing.  In order to indicate a resource is available for indexing it may be annotated with metadata:
+An administrator may wish to restrict indexing activities to particular resources, for example when automatically ingesting data. Should a storage resource be at the edge, that resource may not be appropriate for indexing. In order to indicate a resource is available for indexing it may be annotated with metadata:
 ```
 imeta add -R <resource name> irods::indexing::index true
 ```
-By default, should no resource be tagged it is assumed that all resources are available for indexing.  Should the tag exist on any resource in the system, it is assumed that all available resources for indexing are tagged.
+By default, should no resource be tagged it is assumed that all resources are available for indexing. Should the tag exist on any resource in the system, it is assumed that all available resources for indexing are tagged.
 
-### Plugin Settings
+### Enabling the Indexing Capability Plugins
 
-There are currently three rule engine plugins to configure for the indexing capability which should be added to the `"rule_engines"` section of `/etc/irods/server_config.json`:
+To enable the Indexing capability, prepend the following plugin configuration to the list of rule engines in `/etc/irods/server_config.json`. Plugin-specific configuration options are explained below.
 
-```
+**IMPORTANT: The comments in the JSON structure are there for explanatory purposes and must not be included in your configuration. Failing to follow this requirement will result in the server failing to service requests.**
+
+```js
 "rule_engines": [
     {
         "instance_name": "irods_rule_engine_plugin-indexing-instance",
         "plugin_name": "irods_rule_engine_plugin-indexing",
-        "plugin_specific_configuration": {}
+        "plugin_specific_configuration": {
+            // The lower limit for randomly generated delay task intervals.
+            "minimum_delay_time": 1,
+
+            // The upper limit for randomly generated delay task intervals.
+            "maximum_delay_time": 30,
+
+            // The maximum number of delay rules that are allowed to be scheduled for
+            // collection-based indexing operations.
+            //
+            // If set to 0, the limit is disabled.
+            "job_limit_per_collection_indexing_operation": 1000
```

Comment:
@d-w-moore Does the new wording here align with your understanding of this config option?

@trel Does the new wording clear to you? What do you think it's trying to say?